services/nginx/nginx.conf.template

```diff
@@ -111,7 +111,7 @@ http {
     {% if env.debug %}
     listen 127.0.0.1:{{ ports.app }};
     {% else %}
-    {% if server.ssl %}
+    {% if server.ssl_use_certificate %}
```

Comment:
We don't need this field, I think; `ssl_certificate` is enough by itself.

Reply:
Should be OK? Empty strings are also False.

Reply:
I suppose so, just wondering if this could be the reason why this argument was added. But yes, it should work just fine.